src/js/components/inspectors/FormInputProperty.tsx

```diff
@@ -189,6 +189,9 @@ class BaseFormInputProperty extends React.Component<OwnProps & StateProps & Disp
         );
 
       case 'select':
+        if (value.scheme) {
```

Comment:
Might be a little hacky, open to suggestions on other ways to handle this.  `value = {scheme: "someColorScheme"}` otherwise and threw off the selector options